apps/address-service/domains/common/utils/codegeneration/generate.hooks.ts

```diff
@@ -0,0 +1,220 @@
+import { useCallback } from 'react'
```

Comment:
Can you use `@open-condo/codegen/generate.hooks`?

Reply:
Done